xbmc/cores/VideoPlayer/DVDInputStreams/DVDFactoryInputStream.cpp

```diff
@@ -36,14 +36,25 @@
 #include "storage/MediaManager.h"
 #include "URL.h"
 #include "filesystem/File.h"
+#include "utils/StringUtils.h"
 #include "utils/URIUtils.h"
+#include "addons/VFSEntry.h"
```

Comment:
they do, sort of. i have an add-on adding support for additional .iso files (SACD's). to distinguish these from the video iso images, the content type needs to be set on the item probed. if not it falls back to generic decision based on the fact that the extension is '.iso'.